src/plone/volto/behaviors/preview.py

```diff
@@ -12,7 +12,10 @@ class IPreview(model.Schema):
 
     preview_image = namedfile.NamedBlobImage(
         title=_(u"label_previewimage", default=u"Preview image"),
-        description=u"",
+        description=_(
+            u"help_previewimage",
+            default=u"Insert an image that will override the leadimage in listing blocks.",
```

Comment:
@cekk we should not use the term "leadimage" IMO. It is misleading since it has been used in the past. This is supposed to be the "preview image" which is only used for previews in listings and in teaser elements.

"leadimage" -> "preview image"